WordPress/twentytwentyone
# Fix audio shortcode buttons styling

Fixes https://github.com/WordPress/twentytwentyone/issues/407

## Summary


Excludes the mejs buttons from the button styling

## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:
1. Import the test XML from https://raw.githubusercontent.com/WPTT/theme-unit-test/d12c67cddad161360c88e4df45205b3eded8c93e/themeunittestdata.wordpress.xml
1. Search for the "Post Format: Audio" post
1. Check if it looks okay.


## Quality assurance
